### ISSUE TITLE
Propagation of the click event to the container

### ISSUE THREAD

**Issue by klablink** [RobinHerbots/Inputmask#2423]
I'm trying to port an application from the 3.x version of inputmask to the last one. I have a different behaviour regarding the click event: in the previous version the event was propagated to the container, in the last one it doesn't work. This seems to happen because the handler returns a **false** value; in this case, jquery blocks the event propagation.

```javascript
...
    case "click":
    case "focus":
         ...
         return false;
```

Is it the desired behaviour?

[jsfiddle](https://jsfiddle.net/pbeato/71xseoay/)

- OS: Macos catalina
- Browser: Chrome 86
- Inputmask version: 5.0.6-beta.20

Thanks

**Comment by melloware** [comment]
+1

**Comment by RobinHerbots** [comment]
@klablink , @melloware ,

It should be fixed in Version: 5.0.8-beta.6

**Comment by melloware** [comment]
@RobinHerbots I can confirm my issue is fixed!
